engine/edge.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Docker CE Edge releases
+description: Information and changelogs for Docker CE Edge releasers
+---
+
+## Overview
+
+Docker CE Edge releases are monthly releases cut straight from the `master`
+branch of the Docker repository. These releases allow you to try the latest
+features of Docker in non-production environments. They are effectively previews
```

Comment:
You can run them in production, but won't get fixes after a month. (Which doesn't seem to bother a lot of people running really old versions of docker 😇 )